### DISCUSSION TITLE
posts/nyxbot%E9%83%A8%E7%BD%B2%E6%96%87%E6%A1%A3/

### DISCUSSION THREAD

**Discussion by giscus** [KingPrimes/kingprimes.github.io#2] [Announcements]
# posts/nyxbot%E9%83%A8%E7%BD%B2%E6%96%87%E6%A1%A3/

NyxBot的部署文档

https://kingprimes.us.kg/posts/nyxbot%E9%83%A8%E7%BD%B2%E6%96%87%E6%A1%A3/

**Comment by KingPrimes** [comment]
测试
